utils/constants.js

```diff
@@ -0,0 +1,6 @@
+// pagination
+const PAGE_ITEM = 20;
+const LAST_PAGE = 500;
+const POPULAR = "popular";
+
+export { PAGE_ITEM, LAST_PAGE, POPULAR };
```

Comment:
```suggestion
// pagination
export const PAGE_ITEM = 20;
export const LAST_PAGE = 500;
export const POPULAR = "popular";
```
이렇게 표현하는 방법도 있답니다~!
개인적으로 이렇게 해야 코드를 변경해야 할 일이 생길 때 손이 덜 간다고 생각해요 ㅎㅎ